README.rst

```diff
@@ -0,0 +1,484 @@
+tackle challenging image and video analysis problems and other related
+challenges. Recently started by Kitware’s Computer Vision and
+Scientific Visualization teams, KWIVER is an ongoing effort to
+transition technology developed over multiple years to the open source
+domain to further research, collaboration, and product development.
+
+The project is structured with the parent `kwiver` repository working
+as CMake "super-build" that pulls in a number of KWIVER and other open
+source projects.
+
+
+Building KWIVER
+===============
+
+Fletch
+------
+
+KWIVER has (and will have more) a number of dependencies on 3rd party
+Open Source libraries.  To make it easier to build KWIVER, especially
+on systems like Microsoft Windows that don't have package manager,
+[Fletch](https://github.com/Kitware/fletch) was developed to gather,
+configure and build those packages for use with KWIVER.  Fletch is a
+[CMake](www.cmake.org) based "super-build" that takes care of most of
+the build details for you.
+
+To build Fletch, clone the Fletch repository:
+
+	git clone https://github.com/Kitware/fletch.git
+
+	git submodule update --init
+
+Then, create a build directory and run the following `cmake` command:
+
+	cmake -Dfletch_BUILD_WITH_PYTHON:BOOL=TRUE -DCMAKE_BUILD_TYPE:STRING=Release -Dfletch_ENABLE_Boost:BOOL=TRUE -Dfletch_ENABLE_OpenCV:BOOL=TRUE /path/to/fletch/source/directory
+
+If you have more than one version of Python installed on your system
+and you want to be sure to use a particular one (for example we here
+at KWIVER development central use
+[Anaconda](https://store.continuum.io/cshop/anaconda/) fairly
+frequently) you'll want to add the following arguments to the `cmake`
+command:
+
+* `-DPYTHON_INCLUDE_DIR=/path/to/python/include/directory`  For example, for a default Python 2.7 Anaconda install on Linux this would be `${HOME}/anaconda/include/python2.7`
+* `-DPYTHON_EXECUTABLE=/path/to/executable/python` For example, for a default Python 2.7 Anaconda install on Linux this would `${HOME}/anaconda/bin/python`
+* `-DPYTHON_LIBRARY=/path/to/python/library` For example, for a default Python 2.7 Anaconda install on Linux, this would be `${HOME}/anaconda/lib/libpython2.7.so`
+
+Once your `cmake` command has completed, you can build with the following command
+
+	make
+
+kwiver
+------
+
+Once Fletch has been built, it's possible to build the `kwiver` repository as well.  This repo is also a CMake super-build
+and can be fetched with this command:
+
+	git clone https://github.com/Kitware/kwiver.git
+
+The build can be configured with this command:
+
+	cmake -DKWIVER_ENABLE_PYTHON:BOOL=ON -DKWIVER_ENABLE_ARROWS:BOOL=ON -DKWIVER_ENABLE_PROCESSES:BOOL=ON -DKWIVER_ENABLE_TESTS:BOOL=ON -DKWIVER_ENABLE_TOOLS:BOOL=ON -DKWIVER_USE_BUILD_TREE:BOOL=ON -DKWIVER_ENABLE_VXL:BOOL=ON -DKWIVER_ENABLE_SPROKIT:BOOL=ON -DKWIVER_ENABLE_OPENCV:BOOL=ON -Dfletch_DIR:PATH=/path/to/fletch/build/directory /path/to/kwiver/source/directory
```

Comment:
I would not suggest a specific set of build options here.  Rather talk about some of the high-level build options and what they do, and let the user decide.